builder-support/specs/pdns-recursor.spec

```diff
@@ -75,11 +75,13 @@ make %{?_smp_mflags} check || (cat test-suite.log && false)
 make install DESTDIR=%{buildroot}
 
 %{__mv} %{buildroot}%{_sysconfdir}/%{name}/recursor.conf{-dist,}
+%{__mkdir} %{buildroot}%{_sysconfdir}/%{name}/recursor.d
 
-# change user and group to pdns-recursor
+# change user and group to pdns-recursor and add default include-dir
 sed -i \
     -e 's/# setuid=/setuid=pdns-recursor/' \
     -e 's/# setgid=/setgid=pdns-recursor/' \
+    -e 's!# include-dir=.*!&\ninclude-dir=%{_sysconfdir}/%{name}/recursor.d!' \
```

Comment:
I would prefer not using `&\n` here: `'s!# include-dir=.*!include-dir=%{_sysconfdir}/%{name}/recursor.d!'`

Reply:
Everywhere i see i see Powrdns Auth and one of the db as backend, mysql crash, postgresql crash they all crash in repicas and stand alones in 1 day, 1 week or 1 month, why ? Ive tried mysql, postgres, sqlite which is the only one which fits all the owners criterias. But not Mariadb , redis , but most of all CockroachDB which is twice or 10 times faster, more stable and self healing than postgresql. And Mongodb is never mention.

But i was wondering since all the db we have tried as backends, and almost 80% of all guides say use db as backed, but why cant we use PowerDns as master and 2 or 3 slaves replicating ? That would be substainable or ? Why does it has to be auth with db as backend ? Has anyone ran a master + 2 or 3 powerdns recursors as slave ? And dnsdist is that then needed ? Has powerdns released any panel like knot and nsd?  A Noregian web browsing company has made a GUI for powerdns but it would be nice if they gave us what they use.